samples/app-identity-linking-with-sso/csharp/IdentityLinkingWithSSO/Pages/tab.cshtml

```diff
@@ -381,15 +368,13 @@
                 });
             });
         }
-
         // Get face book profile of user.
         function getServerSideTokenGoogle(clientSideToken) {
             return new Promise((resolve, reject) => {
                 microsoftTeams.getContext((context) => {
-
                     $.ajax({
                         type: 'POST',
-                        url: '/getGoogleAccessToken',
+                        url: '/getGoogleDetailsOauth',
                         dataType: 'json',
```

Comment:
Are we getting OAuth details or User details?
If User details, please rename it.